src/rubysys/encoding.rs

```diff
@@ -100,7 +96,7 @@ extern "C" {
 }
 
 pub unsafe fn coderange_set(obj: Value, code_range: InternalValue) {
-    let basic: *mut RBasic = mem::transmute(obj.value);
+    let basic: *mut RBasic = obj.value as _;
```

Comment:
Can you explain what this change does?

Reply:
Is an equivalent way of achieving the same result, `as` is safer than transmute since `transmute` can also change types of non-pointer types and also lifetimes. In general wherever you can get away with an `as` instead of a transmute we should probably should stick to `as`.